frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineDetails.tsx

```diff
@@ -16,7 +16,10 @@ interface PipelineDetailsProps {
 }
 
 const PipelineDetails: React.FC<PipelineDetailsProps> = ({ obj: pipeline }) => {
-  const routeTemplates: RouteTemplate[] = usePipelineTriggerTemplateNames(pipeline) || [];
+  const {
+    metadata: { name, namespace },
+  } = pipeline;
+  const routeTemplates: RouteTemplate[] = usePipelineTriggerTemplateNames(name, namespace) || [];
```

Comment:
we are already calling `usePipelineTriggerTemplateNames` hook in the parent component `PipelineDetailsPage.tsx`, instead of calling it again here, we can pass the `routeTemplates` data to the child component via `customData` property in the DetailsPage component in the `pipelineDetailsPage.tsx`.
```suggestion

```

Reply:
@karthikjeeyar Done.